src/ctx.cpp

```diff
@@ -425,15 +425,11 @@ void zmq::thread_ctx_t::start_thread (thread_t &thread_,
     thread_.setSchedulingParameters (_thread_priority, _thread_sched_policy,
                                      _thread_affinity_cpus);
 
-    char namebuf[16] = "";
-#ifdef ZMQ_HAVE_WINDOWS
-    LIBZMQ_UNUSED (name_);
-#else
+    char namebuf[128] = "";
```

Comment:
I don't think that this really works. If I am not mistaken, the maximum length is 16 chars on Linux and 32 on Windows. Longer names will be truncated, so it should be ensured that the truncated name stays meaningful.

Reply:
Ok, I'll keep the 32 limit on windows and 16 otherwise.